packages/core/src/components/sections/BannerNewsletter/BannerNewsletter.tsx

```diff
@@ -1,27 +1,38 @@
-import BannerText from 'src/components/ui/BannerText'
-import Newsletter from '../Newsletter'
+import BannerText, { BannerTextProps } from 'src/components/ui/BannerText'
+import Newsletter, { NewsletterProps } from 'src/components/ui/Newsletter'
 import Section from '../Section'
 import styles from './section.module.scss'
 
-function BannerNewsletter() {
+function BannerNewsletter({
+  banner,
+  newsletter,
+}: {
+  banner: BannerTextProps
+  newsletter: NewsletterProps
+}) {
   return (
     <Section
       className={`${styles.section} section-banner-newsletter layout__content`}
     >
       <div data-fs-banner-newsletter>
         <BannerText
-          title="Get to Know Our Next Release"
-          caption="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam elit nisi, vehicula in turpis sit amet, posuere aliquam nisl. "
-          actionLabel="Shop Now"
-          actionPath="/"
-          variant="secondary"
-          colorVariant="light"
+          title={banner.title}
+          caption={banner.caption}
+          actionLabel={banner.actionLabel}
+          actionPath={banner.actionPath}
```

Comment:
I noticed in the `sections.json` fro this section that we receive a `link` prop `text` and `url` instead of `actionLabel` and `actionPath`. Could you double check it?